scout/server/extensions/loqus_extension.py

```diff
@@ -42,95 +42,214 @@ def execute_command(cmd):
 
 class LoqusDB:
     """Interface to loqusdb from Flask
+    Initialized from server/app.py (extensions.loqusdb.init_app(app))
+    Loqus config params should be set in Scout config file
+    """
 
-    NOTE: * initialied in __init__.py called from server/extensins/__init.py__ but also
-            init_app is called from server/app.init.
-          * in practice this is a singleton class
-          * configured in `scout/server/config.py`"""
-
-    def __init__(self, loqusdb_binary=None, loqusdb_config=None, loqusdb_args=None, version=None):
-        """Initialise from args"""
-        self.loqusdb_settings = [
-            {"id": "default", "binary_path": loqusdb_binary, "config_path": loqusdb_args}
-        ]
+    def __init__(self):
+        self.loqusdb_settings = {}
         self.loqus_ids = []
-        self.version = version
```

Comment:
Since loqus is supporting multiple instances I don't think this line has sense any more.